.bazelci/presubmit.yml

```diff
@@ -0,0 +1,51 @@
+---
+bazel: 0.28.1
+tasks:
+  ubuntu1604:
+    name: ":linux:"
+    shell_commands:
+    - wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+    - bash ./Miniconda3-latest-Linux-x86_64.sh -b -p $HOME/miniconda
+    - source $HOME/miniconda/bin/activate
+    - wget https://raw.githubusercontent.com/plaidml/plaidml/master/environment.yml
+    - $HOME/miniconda/bin/conda env update -f ./environment.yml
+    - conda activate $HOME/miniconda
+    build_targets:
+    - "//plaidml/keras:wheel"
+    build_options:
```

Comment:
I think it's called `build_flags` not `build_options`

https://github.com/bazelbuild/continuous-integration/blob/master/buildkite/README.md#using-specific-build--test-flags

Reply:
Fixed in commit [`814cd24`](https://github.com/plaidml/plaidml/pull/435/commits/814cd246c8e9fee003c7abba4b074f7e7dc9259d)